docs2/getting-started.qmd

```diff
@@ -5,8 +5,8 @@ sidebar: getting-started
 
 Welcome to the Ibis project!
 
-- **Learning Ibis for the first time?:** Check out the [Ibis getting started tutorial](./getting_started.md)!
-- **Coming from SQL?**: Take a look at [Ibis for SQL users](./ibis-for-sql-users.ipynb)!
-- **Coming from pandas?**: Check out [Ibis for pandas users](./ibis-for-pandas-users.ipynb)!
-- **Coming from R?**: See [Ibis for dplyr users](./ibis-for-dplyr-users.ipynb)!
+- **Learning Ibis for the first time?:** Check out the [Ibis getting started tutorial](./tutorials/getting_started.qmd)!
+- **Coming from SQL?**: Take a look at [Ibis for SQL users](./tutorials/ibis-for-sql-users.ipynb)!
+- **Coming from pandas?**: Check out [Ibis for pandas users](./tutorials/ibis-for-pandas-users.ipynb)!
+- **Coming from R?**: See [Ibis for dplyr users](./tutorials/ibis-for-dplyr-users.ipynb)!
```

Comment:
I'm not yet sure how to reference notebooks (or the pages that they generate), so these links are still broken.

Reply:
Just needed to be moved to `.qmd`!